src/test/java/com/openlattice/rehearsal/SetupEnvironment.java

```diff
@@ -124,22 +125,22 @@
         String idUser2 = ( String ) jwtUser2.getPrincipal();
         String idUser3 = ( String ) jwtUser3.getPrincipal();
 
-        retrofit.create( PrincipalApi.class ).activateUser();
-        retrofit1.create( PrincipalApi.class ).activateUser();
-        retrofit2.create( PrincipalApi.class ).activateUser();
-        retrofit3.create( PrincipalApi.class ).activateUser();
+        retrofit.create( PrincipalApi.class ).syncCallingUser();
```

Comment:
throwing inside a `clinit` in the jvm causes pretty serious debuggability problems. I'd recommend moving all of this inside a singleton method or instance.